lib/client/config/configHelpers.ts

```diff
@@ -5,6 +5,12 @@ import { getConfig, loadBrokerConfig } from '../../common/config/config';
 import version from '../../common/utils/version';
 import { CONFIGURATION } from '../../common/types/options';
 
+let globalClientOpts: Record<string, any> = {};
```

Comment:
nitpick: we could use unknown instead of any.

Reply:
will fix in the next changes.